gradle/dependency-versions.properties

```diff
@@ -100,8 +100,8 @@ spring-tx.version = 4.3.2.RELEASE
 springframework.version = 4.3.2.RELEASE
 stephenc-findbugs.version = 1.3.9-1
 spymemcached.version = 2.9.0
-swagger.version = 1.3.2
-swagger-springmvc.version = 0.8.2
+swagger.version=1.3.13
```

Comment:
However, 1.3.13 is an old Swagger release. I don't think we should upgrade to an old version. If we're going to upgrade then let's upgrade to the latest version (1.5.10?).